lib/bibliothecary/dependencies_result.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Bibliothecary
+  # DependenciesResult bundles together a list of dependencies and the project name.
+  #
+  # @attr_reader [Array<Dependency>] dependencies The list of Dependency objects
+  # @attr_reader [String,nil] project_name The name of the project
+  class DependenciesResult
```

Comment:
Yeah, ParserResult makes it clearer where its coming from and _why_ it contains what it does.